eva_submission/nextflow/accession.nf

```diff
@@ -224,6 +224,8 @@ process csi_index_vcf {
  * Copy files from eva_public to FTP folder.
  */
  process copy_to_ftp {
+    queue = 'research-rh74'
```

Comment:
I think we should use a [label](https://www.nextflow.io/docs/latest/process.html#label) for this step and set the queue from the config file. This way the code would be compatible to codon with no further changes